app/src/main/java/com/victor/adasfeed/ProfileActivity.kt

```diff
@@ -76,20 +58,44 @@ class ProfileActivity : AppCompatActivity() {
             }
         }
 
+        fabSave = findViewById(R.id.fabSave)
+        fabSave.setOnClickListener { view ->
+            if (editUserName.text.isNotBlank() && editNickname.text.isNotBlank()) {
+                textUserName.text = getString(R.string.profile_name, editUserName.text)
+                textNickname.text = editNickname.text
+                user = user?.copy(
+                    userName = editUserName.text.toString(),
+                    userNickname = editNickname.text.toString()
+                )
+                Snackbar.make(view, "Dados salvos", Snackbar.LENGTH_LONG)
```

Comment:
nit: you could have added the Snackbar message and action to resources

Reply:
True.